### ISSUE TITLE
Error in DRIMSeq::dmDSdata(counts = counts, samples = samps)

### ISSUE THREAD

**Issue by GimenaA** [nf-core/rnasplice#129]
### Description of the bug

I'm getting an error with DRIMSeq when I try to run rnasplice. The pipeline works fine with other samples, so I'm not sure what's causing this error.

### Command used and terminal output

```console
Command Used:
nextflow run nf-core/rnasplice -profile docker -params-file /path/to/params.yaml

Error message:
ERROR ~ Error executing process > 'NFCORE_RNASPLICE:RNASPLICE:DRIMSEQ_DEXSEQ_DTU_SALMON:DRIMSEQ_FILTER (1)'

Caused by:
  Process `NFCORE_RNASPLICE:RNASPLICE:DRIMSEQ_DEXSEQ_DTU_SALMON:DRIMSEQ_FILTER (1)` terminated with an error exit status (1)

Command executed:

  run_drimseq_filter.R salmon.merged.txi.dtu.rds tximport.tx2gene.tsv Arc_RIPseq-samplesheet.csv \
      6 \
      3 \
      3 \
      10 \
      0.1 \
      10
  
  cat <<-END_VERSIONS > versions.yml
  "NFCORE_RNASPLICE:RNASPLICE:DRIMSEQ_DEXSEQ_DTU_SALMON:DRIMSEQ_FILTER":
      r-base: $(echo $(R --version 2>&1) | sed 's/^.*R version //; s/ .*$//')
      bioconductor-drimseq: $(Rscript -e "library(DRIMSeq); cat(as.character(packageVersion('DRIMSeq')))")
  END_VERSIONS

Command exit status:
  1

Command output:
  (empty)

Command error:
  
  Attaching package: 'DRIMSeq'
  
  The following object is masked from 'package:base':
  
      proportions
  
  Error in DRIMSeq::dmDSdata(counts = counts, samples = samps) : 
    all(samples$sample_id %in% colnames(counts)) is not TRUE
  Calls: <Anonymous> -> stopifnot
  Execution halted
```

### Relevant files

Samplesheet file:
sample,fastq_1,fastq_2,strandedness,condition
ID11-IP-Arc-R2,/path/to/rawdata/ID11-IP-Arc-R2_1.fastq.gz,/path/to/rawdata/ID11-IP-Arc-R2_2.fastq.gz,unstranded,treatment
IP-Arc-R1,/path/to/rawdata/IP-Arc-R1_1.fastq.gz,/path/to/rawdata/IP-Arc-R1_2.fastq.gz,unstranded,treatment
ID-2-Arc-IP-R3,/path/to/rawdata/ID-2-Arc-IP-R3_1.fastq.gz,/path/to/rawdata/ID-2-Arc-IP-R3_2.fastq.gz,unstranded,treatment
ID-5-Non-IP-R3,/path/to/rawdata/ID-5-Non-IP-R3_1.fastq.gz,/path/to/rawdata/ID-5-Non-IP-R3_2.fastq.gz,unstranded,Control
ID12-Non-Arc-R2,/path/to/rawdata/ID12-Non-Arc-R2_1.fastq.gz,/path/to/rawdata/ID12-Non-Arc-R2_2.fastq.gz,unstranded,Control
Non-Arc-R1,/path/to/rawdata/Non-Arc-R1_1.fastq.gz,/path/to/rawdata/Non-Arc-R1_2.fastq.gz,unstranded,Control

params.yaml file:
input: '/path/to/sequencing_data/Arc_RIPseq-samplesheet.csv'
contrasts: '/path/to/sequencing_data/Arc_RIPseq-contrastsheet.csv'
outdir: '/path/to/sequencing_data/'
fasta: '/home/gimena/reference_files/BDGP6.32_ensemble/Drosophila_melanogaster.BDGP6.32.dna.toplevel.fa'
gtf: '/home/gimena/reference_files/GTF_files/fromEnsembl/Drosophila_melanogaster.BDGP6.32.106.gtf'
source: 'fastq'
aligner: 'star_salmon'
min_samps_gene_expr: 6
min_gene_expr: 10
min_samps_feature_expr: 3
min_feature_expr: 10
min_samps_feature_prop: 3
min_feature_prop: 0.1
sashimi_plot: true
fig_height: 20
fig_width: 20
miso_genes: FBgn0265487
skip_bigwig: false
dexseq_dtu: true
suppa: false
edger_exon: false
dexseq_exon: false
rmats: false

### System information

Nextflow Version: version 23.10.0
Linux OS
Version of nf-core/rnasplice 1.0.1

**Comment by pothepanda09** [comment]
I'm also running into the same issue! I thought it was the minimum sample/feature tags, so I set it to zero:
min_samps_gene_expr: 0
min_gene_expr: 0
min_samps_feature_expr: 0
min_feature_expr: 0
min_samps_feature_prop: 0
min_feature_prop: 0
That wasn't the issue either, so I'm hoping someone knows how to solve this.

**Comment by jma1991** [comment]
Hey @GimenaA 

Thanks for reporting your issue. Can you please try with the latest release? I have a feeling you are using an older version where we didn't handle sample names with dashes in them correctly. See this [PR](https://github.com/nf-core/rnasplice/pull/124) for details.

**Comment by jma1991** [comment]
Closing as completed. Please re-open if you are still facing the same issue.

**Comment by esamorodnitsky** [comment]
I am encountering this same issue and reopened it.

**Comment by piplus2** [comment]
Fixed by #141
